express/code/scripts/utils/ratings-utils.js

```diff
@@ -21,7 +21,7 @@ const RNR_API_URL = isProd ? 'https://rnr.adobe.io/v1' : 'https://rnr-stage.adob
 
 // Errors, Analytics & Logging
 const lanaOptions = {
-  sampleRate: 100,
+  sampleRate: 10,
```

Comment:
i believe the default sampleRate for most use cases was 1%? Or was it updated to be 10% now?